### DISCUSSION TITLE
Gatsby Admin

### DISCUSSION THREAD

**Discussion by mxstbr** [gatsbyjs/gatsby#27402] [Umbrella Discussions]
People experimenting with Admin — this is the place to ask questions, report bugs, suggest ideas and collaborate on improvements.

**Quick links:**

- [Docs](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-admin/README.md)
- [Code](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-admin/src)

**Known issues:**

- Some `gatsby-config.js` formats are not supported. If your `gatsby-config.js` is not supported, we'd love if you could share what it looks like below!

**Comment by onurtahmaz** [comment]
Just tried to install gatsby-plugin-mdx-embed via Gatsby Admin which is a great idea btw! After installation, there was no problem in the gatsby-config.js; however in package.json, mdx-embed package must be installed for the former to work. 

So; I think the problem is the fact that Gatsby Admin runs the following command: `npm install gatsby-plugin-mdx-embed`; instead of `npm install gatsby-plugin-mdx-embed mdx-embed --save` as stated in the plugin's documentation.

**Comment by crock** [comment]
Getting the following error when I try to setup gatsby-admin for development using the yarn workspace command listed in the readme. I've forked and cloned the entire gatsby repo in a GH Codespace and installed the dependencies.
![image](https://user-images.githubusercontent.com/8924090/96137241-6a07a280-0eca-11eb-8b36-3af7cacaf6af.png)

**Comment by mxstbr** [comment]
@crock the guide in the gatsby-admin README assumes you already have the gatsby repo set up locally, I think you haven't run `yarn bootstrap` yet? 🤔

**Comment by crock** [comment]
> @crock the guide in the gatsby-admin README assumes you already have the gatsby repo set up locally, I think you haven't run `yarn bootstrap` yet? 🤔

Ah thanks, that's what I was missing.

**Comment by dedevillela** [comment]
Hi all. Loving it so far! Thanks for this great app. :)
I've found some issues while browsing plugins, looks like it only renders properlly formated .md files as the README, the ones with HTML aren't good to read...
![gatsby-source-wordpress-experimental](https://user-images.githubusercontent.com/3102096/96261169-72c5aa80-0f96-11eb-94f3-25f07c478733.png)

Also, make sure ALL links in plugin's README open in new tab/window with default browser (instead of inside the app itself! 👎 ), jut like the "View localhost" and "View on GitHub" buttons. 🥇

I'm running on Mac OS High Sierra, btw.

<img width="698" alt="highsierra" src="https://user-images.githubusercontent.com/3102096/96351672-75f29080-1093-11eb-9b41-711277e5944f.png">

Best regards,
Andre
@dedevillela

_Originally posted by @dedevillela in https://github.com/gatsbyjs/desktop/issue_comments/710030742_ &
_Originally posted by @dedevillela in https://github.com/gatsbyjs/desktop/issue_comments/710062663_

**Comment by Anmol368** [comment]
riginally posted by @dedevillela in https://github.com/gatsbyjs/desktop/issue_comments/710030742

**Comment by giacomocerquone** [comment]
bug here. Hovering on the "manage" link it destroys the page styles. (I've reported it here initially https://github.com/gatsbyjs/desktop/issues/50)

![107286570-0bf66080-6a61-11eb-9c0c-9733cb12c26c](https://user-images.githubusercontent.com/9303791/107288648-12d2a280-6a64-11eb-8dd9-cc6dc1de71dd.png)
